src/pages/index.tsx

```diff
@@ -36,12 +37,24 @@ const Homepage = () => {
   });
   const match = api.match.getOngoingMatch.useQuery();
   const matchStatus = match.data && match.data !== EMPTY_RES;
+  const [disabled, setDisabled] = useState(
+    loggedUser.isLoading || match.isLoading,
+  );
+
+  useEffect(() => {
```

Comment:
What about simplifying this a bit. Having conditionals to set booleans is a bit of a code smell. 

`setDisabled(loggedUser.isLoading || match.isLoading)`

OR

```
const isDisabled = loggedUser.isLoading || match.isLoading
setDisabled(isDisabled)
```